mysql/assets/configuration/spec.yaml

```diff
@@ -66,7 +66,12 @@ files:
               If you specify a socket you dont need to specify a host.
             value:
               type: string
-
+          - name: charset
+            description: Charset you want to use.
+            value:
+              type: string
+              example: utf8
+              default: None
```

Comment:
Should we specify `default` as the value `pymysql` use by default? I'm not sure what would be clearer for the customer.

Reply:
We should not specify a default unless its our default because it can change under the hood